### PR TITLE
Update snapshot CR name to "snap-"<uuid> format.

### PR DESCRIPTION
**Pull request by swgupta-vmw**
Signed-off-by: Swati Gupta <swgupta@swgupta-a01.vmware.com>

### CODE REVIEW

**Comment by svcbot-qecnsdp** [comment]
Congratulations !! 

All the pre-check tests passed
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/78/

File: pkg/snapshotUtils/snapshot.go [hunk 1]
```diff
@@ -48,39 +50,31 @@ func checkPhasesAndSendResult(waitForPhases []backupdriverv1.SnapshotPhase, snap
...
+// Create a Snapshot record in the specified namespace.
+func SnapshotRef(ctx context.Context,
+	clientSet *v1.BackupdriverV1Client,
+	objectToSnapshot core_v1.TypedLocalObjectReference,
+	namespace string,
+	repository BackupRepository,
+	waitForPhases []backupdriverv1.SnapshotPhase,
+	logger logrus.FieldLogger) (backupdriverv1.Snapshot, error) {
 
+	snapshotUUID, err := uuid.NewRandom()
```
Comment (xing-yang) [inline pkg/snapshotUtils/snapshot.go:62]: So snapshotUUID is still generated inside SnapshotRef.  How do you make it is not generated again for the same request?

Comment (swgupta-vmw) [reply pkg/snapshotUtils/snapshot.go:62]: That was fixed by Lintong's commit - https://github.com/vmware-tanzu/velero-plugin-for-vsphere/commit/865897be99637a8626714346c922df5f0fe0b174

**Comment by svcbot-qecnsdp** [comment]
Congratulations !! 

All the pre-check tests passed
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/84/

**Pull request merged**